scripts/namespace_codegen_templates.sh

```diff
@@ -17,6 +17,7 @@ cp -r "$codegenDir/templates" /tmp/namespaced-templates
 find /tmp/namespaced-templates/ -type d -print0 | xargs -0 chmod 755
 
 # Rename the "runtime" package to "codegen_runtime" in all templates.
-find /tmp/namespaced-templates -type f -print0 | xargs -0 sed -i -E 's/\bruntime\b/codegen_runtime/g'
+# It happens to be always used as " runtime.", so find that. "\b" doesn't work in POSIX sed (which OS X uses).
+find /tmp/namespaced-templates -type f -print0 | xargs -0 sed -i -E --posix 's/ runtime./ codegen_runtime./g'
```

Comment:
blegh turns out mac `sed` doesn't support the `--posix` flag

```
sed: illegal option -- -
usage: sed script [-Ealnru] [-i extension] [file ...]
	sed [-Ealnu] [-i extension] [-e script] ... [-f script_file] ... [file ...]
```

Reply:
Blerg. It would be nice to not have two versions of the command.
What if you just rip out `--posix`, does that work for you?  I only added it to guard me (on linux) from using non-posix extensions. But it works for me without `--posix` too.

> it would be a somewhat cumbersome to require all osx devs to do so (install gnu-sed).
+1

Reply:
Ah that worked! Removed and updated